runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -189,6 +190,11 @@ public void onSetMessage(SetMessage setMessage)
 			log.debug("Running total level lookup");
 			executor.submit(() -> playerSkillLookup(setMessage, "total"));
 		}
+		else if (config.lvl() && message.toLowerCase().equals("!cmb"))
+		{
+			log.debug("Running combat level lookup");
```

Comment:
Doesn't `HighscoreClient.lookup` already log when performing a lookup?

Reply:
It tells you it build an URI, but not why. Every other other command specify what command was run

Reply:
Ah, that's fine then I guess. :woman_shrugging: